pkg/ddevapp/router.go

```diff
@@ -16,12 +17,23 @@ import (
 	"github.com/ddev/ddev/pkg/globalconfig"
 	"github.com/ddev/ddev/pkg/netutil"
 	"github.com/ddev/ddev/pkg/nodeps"
+	"github.com/ddev/ddev/pkg/output"
 	"github.com/ddev/ddev/pkg/util"
 	dockerTypes "github.com/docker/docker/api/types"
 )
 
 // RouterProjectName is the "machine name" of the router docker-compose
-const RouterProjectName = "ddev-router"
+const (
+	RouterProjectName     = "ddev-router"
+	MinEphemeralHTTPPort  = 50080
+	MaxEphemeralHTTPPort  = 50442
+	MinEphemeralHTTPSPort = 50443
+	MaxEphemeralHTTPSPort = 50500
```

Comment:
There's an unfortunate assumption here that the HTTP and HTTPS ports are the only thing we're dealing with, when in reality there may be many ports, supported by HTTP_EXPOSE or HTTPS_EXPOSE in extra services. (We don't have a problem with the database port, which is not handled by the router, but many other ports are)